specs/plugin-extensibility.md

```diff
@@ -380,24 +380,24 @@ All request attributes are required.
   "payloadType" : "application/vnd.cncf.notary.payload.v1+json",
   
   // The expected response signature envelope
-  "signatureEnvelopeType" : "application/vnd.cncf.notary.v2.jws.v1"
+  "signatureEnvelopeType" : "application/jose+json"
 }
 ```
 
 *keyId* : Required field that has the key identifier (`keyId`) associated with signing key `keyName` in `config.json`.
 
 *pluginConfig* : Optional field for plugin configuration. For details, see [Plugin Configuration section](#plugin-configuration).
 
-*signatureEnvelopeType* - defines the type of signature envelope expected from the plugin. As  Notation clients need to be updated in order to parse and verify new signature formats, the default signature format can only be changed with new major version releases of Notation. Users however can opt into using an updated signature format supported by Notation, by passing an optional parameter.
-e.g. `notation sign $IMAGE --key {key-name} --signatureFormat {some-new-format}`
+*signatureEnvelopeType* - defines the type of signature envelope expected from the plugin. As  Notation clients need to be updated in order to parse and verify new signature formats, the default signature format can only be changed with new major version releases of Notation. Users however can opt into using an updated signature envelope format supported by Notation, by passing an optional parameter.
+e.g. `notation sign $IMAGE --key {key-name} --envelope-type {some-new-type}`
```

Comment:
IMO, you want to retain "signature-format", "envelope type" is too abstract for users.

Reply:
@chloeyin @shizhMSFT @priteshbandi @yizha1 , can we consider renaming this to some thing more user friendly.

Reply:
> IMO, you want to retain "signature-format", "envelope type" is too abstract for users.

@dtzar Any comments from your side changing `envelope-type` to `envelope-format`?

Reply:
+1 to keep using `envelope-format`. It will be more consistent with the term `envelope-format` we used in the [signature-specification.md](https://github.com/notaryproject/notaryproject/blob/main/signature-specification.md#signature-envelope). 

The flag name `envelop-type` should also be updated in Notation CLI if we agree here.